src/records/RecCore.cc

```diff
@@ -244,7 +244,7 @@ RecCoreInit(Diags *_diags)
     RecLog(DL_Note, "records parsing completed.");
     if (!err.empty()) {
       std::string text;
-      RecLog(DL_Note, "%s", swoc::bwprint(text, "{}", err).c_str());
+      Warning("%s", swoc::bwprint(text, "We have found the following issues when reading the records node:\n {}", err).c_str());
```

Comment:
Beyond the wording change, the move from RecLog to Warning will change where the message is printed?

Reply:
It will go to where we bind `stderr` to. `RecLog(DL_Warning, ...` could also be used I guess. Base on my tests destination should not change.